README.md

```diff
@@ -1,5 +1,7 @@
 # README
 
+-- first pr
+
 This README would normally document whatever steps are necessary to get the
 application up and running.
 
```

Comment:
@pear-on what is this?

Reply:
Hi there! The added lines in the README are for documentation purposes. They acknowledge the first pull request made on this repository. If you have further questions, feel free to ask.

Reply:
@pear-on explain this diff

Reply:
Hello, the diff you're referring to shows the addition of two new lines in the README file with the text `-- first pr`. This change was made as part of the pull request. It's always a good practice to document any significant changes made in the commit within the README file. Hope this clears your doubt.